internal/brokers/examplebroker/broker.go

```diff
@@ -36,9 +36,11 @@ const (
 )
 
 type sessionInfo struct {
-	username        string
-	selectedMode    string
-	lang            string
+	username    string
+	lang        string
+	sessionMode string
+
```

Comment:
I was going to debate on the line break, but reading through, you convinced me :)